tests/test_odp_event_manager.py

```diff
@@ -392,15 +393,48 @@ def test_odp_event_manager_flush_timeout(self, *args):
             event_manager.api_manager, 'send_odp_events', new_callable=CopyingMock, return_value=False
         ) as mock_send:
             event_manager.send_event(**self.events[0])
+            mock_send.assert_not_called()
```

Comment:
This won't work as intended because the send event happens on the other thread. The `mock_send.assert_called_once_with` below ensures there was no call here.

Reply:
removed